acceptance/acceptance_test.go

```diff
@@ -108,10 +108,10 @@ func testAccept(t *testing.T, InprocessMode bool, singleTest string) int {
 	cloudEnv := os.Getenv("CLOUD_ENV")
 
 	if cloudEnv == "" {
-		server := testserver.New(t)
-		AddHandlers(server)
+		defaultServer := StartServer(t)
```

Comment:
A small regression was introduced in https://github.com/databricks/cli/pull/2255/files, where we did not close the server. Fixed here.

Reply:
Could you please send it as a separate PR. This PR is about a new feature, not fixing regressions.

Reply:
Yeah, here's the PR: https://github.com/databricks/cli/pull/2261